tests/unit/dstoreMemory/Test.js

```diff
@@ -93,22 +92,24 @@ define([
 						"element[3].innerHTML should be Selection 4");
 					assert.strictEqual(dstoreMemorylist1Elements[4].innerHTML, "Selection 5",
 						"element[4].innerHTML should be Selection 5");
-
-				});
+					d.resolve();
```

Comment:
If I don't have the d.resolve() it does not resolve, it will timeout.  I also tried to pass a function into callback(), but I think I am going to go back to the way things are done in the other tests, this was just a test to see if I could get the async working.